src/event.c

```diff
@@ -225,6 +233,22 @@ static void handle_button_action(int act)
 	case BNACT_SENS_RESET:
 		cfg.sensitivity = 1.0f;
 		break;
+	case BNACT_FILTER_ROTATION:
+		if (val == KEY_RELEASE)
```

Comment:
formatting fix: the convention for this project is:
  - opening curly braces on the same line as the statement.
  - no space between if and open parenthesis.

Reply:
ok